gallery/container/insights-Microsoft.RedHatOpenShift-OpenShiftClusters.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
the odd thing here though:  the workbook type is `container-insights`, not `container`

so this folder should be container-insights, and the file names should be `[resourcetype].json`

Reply:
Good catch, the script I wrote to convert the gallery files incorrectly assumes the prefix before the first '-' is the workbook type. I'll fix that.